content/en/events/2024/kcsna/registration.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Registration"
+type: docs
+weight: 2
+---
+
+Registration is open to all members of one of the [Kubernetes orgs], qualified Etcd contributors, or sponsored attendees.
+Registration closes: **Thursday, November 7th, 2023; 16:00 US Eastern Time**
+
+If you are not yet a member, but active within the project, please consider
+joining! The process of joining is straight forward. See our [org membership]
+docs for more information. If your application to join is pending, please
+contact us at summit-team@kubernetes.io about registering.
+
+
+
+**NOTE:** The summit adheres to KubeCon’s [Health & Safety Guidelines]
+
+If you acknowledge the above, please go ahead and click below to register:
+
+<h3>
+<a href="https://cvent.me/Ak8z00?locale=en-US&tm=5BFaB2eHY0xZyx_vVZdfLzrXFWPJJsz___EvVtztajE&rp=D599C1DE-92F0-4EC4-8DA2-82B18574E7EF" rel="noopener noreferrer" target="_blank">Register Here</a>
+</h3>
```

Comment:
Should replace with 'registration will open soon' for now. That links to an old dead reg page.

More so a note for ourselves when we DO add the reg links. We need to make it super explicit that non-org members MUST have a sig lead reach out to us before they register. Or whatever process we want to do. Just not have them + a lead into the convo >_>